src/bin/vip-import.js

```diff
@@ -18,3 +18,12 @@ command( )
 	.argv( process.argv, async () => {
 		await trackEvent( 'vip_import_command_execute' );
 	} );
+
+command( )
+	.command( 'media', 'Import media files to your application from a compressed archive' )
+	.command( 'media-progress', 'Import media files to your application from a compressed archive' )
```

Comment:
I think we should follow the convention started by the SQL import here. Instead of having a separate command, let's have a nested command like so : https://github.com/Automattic/vip/blob/527a615b661c70ecdd8ba3f6122476767c2d9420/src/bin/vip-import-sql.js#L361-L361

Also, maybe just `status` , again to align with SQL imports.

Same goes for the validate files command below.

Reply:
I agree, I guess this is an oversight because that was the initial intention.